### ISSUE TITLE
Update image to use gcr.io/distroless/static:nonroot as base for final stage

### ISSUE THREAD

**Issue by krancour** [brigadecore/brigade-bitbucket-gateway#65]

**Comment by hamees-sayed** [comment]
```
FROM alpine:3.7

RUN apk update && apk add --no-cache \
    ca-certificates \
    git \
    && update-ca-certificates

COPY rootfs/brigade-bitbucket-gateway /usr/bin/brigade-bitbucket-gateway
COPY rootfs/gitssh.sh /gitssh.sh
``` 

to this right? 

```
FROM gcr.io/distroless/static:nonroot as final

RUN apk update && apk add --no-cache \
    ca-certificates \
    git \
    && update-ca-certificates
```

**Comment by krancour** [comment]
@hamees-sayed this was fixed already in #65. The issue didn't auto-close because the target branch is `v2` and not `main`. (Auto-closing only works on the main branch.)

Sorry about the confusion.
